controlplane/internal/auth/auth.go

```diff
@@ -0,0 +1,25 @@
+package auth
```

Comment:
I was referring to the same `controller` package but I guess this would work as well

Reply:
Oh oops, do you have a preference?